www/plas02/styless_ukol4.css

```diff
@@ -0,0 +1,127 @@
+*{
+    font-family: sans-serif;
+}
+
+.red_text {
+    color: red;
+}
+
+.green_text {
+    color: rgb(47, 235, 41);
+}
+
+.blue_text {
+    color: #3399ff;
+    font-weight: bold;
+}
+
+.small_text {
+    font-size: 18px;
+}
+
+.big_text {
+    font-size: 24px;
+    font-style: italic;
+}
+
+.background_text {
+    background: brown;
+}
+
+.small_p{
+    font-size: 32px;
+    text-align: center;
+}
+
+.capital_p {
+
+    text-align: justify;
+    text-transform: uppercase; 
+}
+
+.white_box {
+    height: 300px;
+    width: 400px;
+    border-top: 1px solid purple;
+    border-right: 1px solid purple;
+    border-bottom: 1px solid purple;
+    border-left: 1px solid purple;
+    padding: 30px;
+}
+
+.red_box {
+    height: 200px;
+    width: 500px;
+    background: red;
+    margin: 100px;
+}
+
+.invisible_text{
+    color: white;
```

Comment:
tady jsem chtel visibility: hidden, protoze co kdyz se zmeni barva pozadi, to bile pismo bude videt ne?